doc/user/content/sql/show-connections.md

```diff
@@ -25,8 +25,8 @@ SHOW CONNECTIONS;
 ```
 
 ```nofmt
-       name
----------------------
+       name          | type
+---------------------+------
  kafka_connection
```

Comment:
The data rows here are missing their types, no?

Reply:
Yep, just fixed it.